src/region/RegionAnnotation.tsx

```diff
@@ -16,10 +17,32 @@ const RegionAnnotation = (props: Props, ref: React.Ref<HTMLAnchorElement>): JSX.
     const { isActive, shape, ...rest } = props;
     const { height, width, x, y } = shape;
     const className = classNames('ba-RegionAnnotation', { 'is-active': isActive });
+    const targetRef: React.Ref<SVGRectElement> = React.useRef(null);
+
+    React.useEffect(() => {
```

Comment:
Should this logic live in `AnnotationTarget`? We're going to want the same behavior for all annotation types, at least if/until we support non-document file types.